software/discourse.md

```diff
@@ -0,0 +1,22 @@
+# Discourse
+
+We use [Discourse](https://discourse.freistil.it/) for internal discussions that go beyond the context and/or scope of [Asana](/software/asana.html) and [Slack](/software/slack.html).
+
+Since Asana has it's limitations when it comes to quoting text and threaded replys (those simply don't exist there), for more complex discussions we have decided to use discourse, as it is specifically designed to handle long-term discussions with multiple partners and a potentially complex reply structure.
+
+Slack comes with it's own caveats - while it offers a single reply-thread per initial post there is no more complexity available beyond that, and additionally Slack does not lend itself well to conversations that are expected to be carried on over the current day, as slack is a more or less synchronous medium.  
```

Comment:
Use a double hyphen `--` to get an [em dash](http://www.thepunctuationguide.com/em-dash.html).